python/sglang/srt/entrypoints/openai/protocol.py

```diff
@@ -126,6 +126,9 @@ class StructuralTagResponseFormat(BaseModel):
     structures: List[StructuresResponseFormat]
     triggers: List[str]
 
+class StructuralTagResponseFormatV1(BaseModel):
+    type: Literal["structural_tag"]
+    format: Dict[str, Any]
```

Comment:
You can just import StructuralTag from xgrammar, that's also a pydantic BaseModel and all its fields are type checked.